### ISSUE TITLE
You shouldn't be able to upvote yourself

### ISSUE THREAD

**Issue by TheNain38** [HelloZeroNet/ZeroTalk#1]

**Comment by HelloZeroNet** [comment]
While it could be filtered on client-side, but for example facebook or disqus also allows to like your own comment/post.

Maybe setting the upvote to 0 by default would be more better solution

**Comment by TheNain38** [comment]
I'm not using facebook because of privacy issues, but I didn't know that they allow upvoting yourself. So yes setting upvote to 0 by default would be better

**Comment by christopherturner** [comment]
@TheNain38 Yeah, it would make the most sense to set default upvote to 0 and continue allowing users to upvote their own content.  Also, just bumping this issue.  :+1:

**Comment by Angristan** [comment]
Agreed :+1:

**Comment by Codes4Fun** [comment]
The upvote should be by default, that is how reddit does it, and I would suspect most users would just upvote their own post anyway, it also makes it easier to find your own posts since you will see the green vote counter.

**Comment by fiskr** [comment]
@Codes4Fun it sounds like there are two separate features, then: 

0. Set the default count to 0.
0. Automatically upvote yourself, after posting a comment or thread.

**Comment by fiskr** [comment]
Here are the bits of logic to update to set the default count back to 0.
Just remove the `+1` from these subqueries:

- [Line 66 from TopicShow.coffee](https://github.com/HelloZeroNet/ZeroTalk/blob/master/js/TopicShow.coffee#L66)
- [Line 144 from TopicShow.coffee](https://github.com/HelloZeroNet/ZeroTalk/blob/master/js/TopicShow.coffee#L144)
- [Line 135 from TopicList.coffee](https://github.com/HelloZeroNet/ZeroTalk/blob/master/js/TopicList.coffee#L135)
- [Line 162 from TopicList.coffee](https://github.com/HelloZeroNet/ZeroTalk/blob/master/js/TopicList.coffee#L162)

**Comment by imachug** [comment]
@HelloZeroNet Any thoughts on this?
